src/components/FormInput/index.tsx

```diff
@@ -0,0 +1,26 @@
+import styled from "styled-components";
+
+function FormInput() {
+  return (
+    <>
+      <StyledFormInput type="text" />
+    </>
+  );
```

Comment:
```suggestion
  return (
      <StyledFormInput type="text" />
  );
```
여기 부분 작성하신 이유가 있을까요?

Reply:
이 부분은 습관처럼 되서 저렇게 쓰는 방법은 안 좋은가요?!

Reply:
안 좋다기보단 저렇게 쓰는 이유가 return 할 때 하나의 태그로 묶기 위해서 쓰는 건데 이 FormInput은 굳이 감쌀 이유가 없는 것 같아요